lib/internal/modules/esm/translators.js

```diff
@@ -315,9 +287,10 @@ translators.set('commonjs-sync', function requireCommonJS(url, source, isMain) {
   initCJSParseSync();
   assert(!isMain);  // This is only used by imported CJS modules.
 
-  return createCJSModuleWrap(url, source, isMain, (module, source, url, filename) => {
+  return createCJSModuleWrap(url, source, isMain, (module, source, url, filename, isMain) => {
     assert(module === CJSModule._cache[filename]);
-    CJSModule._load(filename);
+    assert(!isMain);
+    CJSModule._load(filename, null, isMain);
```

Comment:
If `isMain` is guaranteed to be falsy, is there any reason to pass it to `CJSModule._load`?

Reply:
Is there a reason *not* to pass it explicitly and rely on the implicit default value of CJSModule._load even though the value is readily available?